cmd/api-response.go

```diff
@@ -947,19 +947,10 @@ func writeSuccessResponseHeadersOnly(w http.ResponseWriter) {
 // writeErrorResponse writes error headers
 func writeErrorResponse(ctx context.Context, w http.ResponseWriter, err APIError, reqURL *url.URL) {
 	switch err.HTTPStatusCode {
-	case http.StatusServiceUnavailable:
+	case http.StatusServiceUnavailable, http.StatusTooManyRequests:
```

Comment:
isn't retry after 60s too long?

we still issue "Deadline exceeded while waiting in incoming queue, please reduce your request rate" error if max requests exceeded - can be reworded since we no longer use a deadline

Reply:
oh yes

Reply:
its only a hint, its upto the client to honor it @poornas